_data/projects/seneca.yml

```diff
@@ -8,5 +8,5 @@ tags:
 - api
 - microservices
 upforgrabs:
-  name: New Contributor
-  link: https://github.com/issues?q=is%3Aopen+is%3Aissue+user%3Asenecajs+label%3A%22new+contributor%22+
+  name: Review
+  link: https://github.com/issues?q=is%3Aopen+is%3Aissue+user%3Asenecajs
```

Comment:
@KuroKen91 https://github.com/senecajs/seneca/labels/LABEL_NAME is preferred over other links. Please create a new label under https://github.com/senecajs/seneca/labels and add that here or use any existing ones from there.

Reply:
Requested changes have been applied. However, it seems the repo is active and the labels are changing quite frequently. To keep this open, I referenced the Doc tag as it may be the easiest for new members to contribute toward.

Reply:
Cool, thanks